middlewares/validators/validator.helper.js

```diff
@@ -777,7 +791,7 @@ function searchSortValidator(fieldLocation, fieldName) {
             } else {
                 return false;
             }
-            if (!!model.searchableField(value)) {
+            if (model.searchableField(value)) {
```

Comment:
Was this a boolean cast? `!!` is odd in an if, should be casting already, just want to make sure we don't break anything here

Reply:
not sure, i think the lint took it off by itself, should i restore it to !!?